src/ensembl/src/content/app/browser/drawer/drawer-views/transcript-summary/TranscriptSummary.tsx

```diff
@@ -156,21 +158,27 @@ const GENE_AND_TRANSCRIPT_QUERY = gql`
 
 const TranscriptSummary = () => {
   const ensObjectGene = useSelector(getBrowserActiveEnsObject) as EnsObjectGene;
+  const [shouldShowDownload, showDownload] = useState(false);
 
-  const transcriptTrack = ensObjectGene?.track?.child_tracks?.[0];
+  const { data: geneData } = useGetTrackPanelGeneQuery({
```

Comment:
Would it be better to name it something like `useGetGeneDataForTrackPanel` instead?